### PR TITLE
Package generator: Search and replace package name

### PR DESCRIPTION
**Pull request by leogermani**
#### Changes proposed in this Pull Request:

* Searches and automatically replace the project name in skeleton files

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack product discussion

NA
#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* Run `jetpack generate` to generate a new package
* Look at the new package created and confirm the `@package` annotation in `src/example.php` file has the right package name
* Also confirm that three values in `.phpcs.dir.xml` file have the right package name

### CODE REVIEW

File: tools/cli/commands/generate.js [hunk 1]
```diff
@@ -266,6 +266,7 @@ export async function generateProject(
 
 	switch ( answers.type ) {
 		case 'package':
+			await searchReplaceInFolder( projDir, 'PACKAGE-NAME', normalizeSlug( answers.name ) );
```
Comment (anomiex) [inline tools/cli/commands/generate.js]: CI doesn't seem to like the uppercase "PACKAGE-NAME", and I don't see any instances of "package-name". Should be safe to just go with that instead.

I wonder if we should put this call up near line 252 instead though.

Comment (leogermani) [reply tools/cli/commands/generate.js]: There's a lower case `package-name` in `.phpcs.dir.xml`.

We can change example.php to lowercase as well

Comment (leogermani) [reply tools/cli/commands/generate.js]: > I wonder if we should put this call up near line 252 instead though.

You mean doing it for all project types?

Comment (leogermani) [reply tools/cli/commands/generate.js]: and there are some upper-case in example.php

Comment (anomiex) [reply tools/cli/commands/generate.js]: > There's a lower case package-name in .phpcs.dir.xml.

You mean the ones you're trying to make uppercase in this PR, that are making phpcs unhappy?

> You mean doing it for all project types?

Yes. Why not?

> and there are some upper-case in example.php

We're not replacing those currently, although we probably should. They could be lowercased too, or we could replace both upper- and lower-case versions of the string.

Comment (leogermani) [reply tools/cli/commands/generate.js]: updated

**Review by anomiex — APPROVED** [review]

**Pull request merged**